### ISSUE TITLE
Customize overall icons

### ISSUE THREAD

**Issue by stefann01** [ant-design/ant-design#50458]
### What problem does this feature solve?

End-user experience could be improved if icons for following components could be replaced: Select, Time / Date Pickers, Pagination etc. any component that has icon

### What does the proposed API look like?

I guess the components could have icon properties such as: customIcon: React.ReactNode

**Comment by zystudios** [comment]
I agree with this, all components with icons should support custom replacements to be able to do this

**Comment by ilxqx** [comment]
这个问题有进展吗？

**Comment by afc163** [comment]
Duplicate of https://github.com/ant-design/ant-design/issues/41413
